update the description of the annotated_arg/ptr spec (#1)
tiwaria1/llvm: sycl/doc/extensions/proposed/sycl_ext_oneapi_annotated_arg.asciidoc

template <typename T, typename PropertyListT = properties<>>
class annotated_arg {
  public:
    annotated_arg() noexcept;
    annotated_arg(const T& v_, const properties<PropertyListT> &P = properties{}) noexcept;
    template<typename T, typename... PropertyValueTs>

>     annotated_arg(T *Ptr, PropertyValueTs... props) noexcept;

Reviewer: ```suggestion
    annotated_arg(const T& v, PropertyValueTs... props) noexcept;
```

a|
[source,c++]
----
annotated_arg(const T& v_, const properties<PropertyListT> &P = properties{}) noexcept;
----
| Not available in device code.
Constructs an `annotated_arg` object from the input object `v_`.

The new property set `PropertyListT` contains all properties in `P`.
If there were duplicate property in `P`, the value of this property must be same.

// --- ROW BREAK ---
a|
[source,c++]
----
template<typename T, typename... PropertyValueTs>
annotated_arg(const T& v_, const PropertyValueTs... props) noexcept;
----
|
Constructs an `annotated_arg` object from the input object `v_`. 

The new property set `PropertyListT` contains all properties listed in `props`.

> If there were duplicate property in the list of `props`, the value of this property must be same.

Reviewer: ```suggestion
If there are duplicate properties present in `props`, the values of the duplicate properties must be the same.
```
And the same wording correction in the descriptions below.
